src/main.rs

```diff
@@ -16,19 +16,22 @@ mod snapshot_tests;
 fn main() {
     human_panic::setup_panic!();
 
-    let Cargo::SemverChecks(args) = Cargo::parse();
+    let Cargo::SemverChecks(mut args) = Cargo::parse();
+
+    let feature_flags = HashSet::from_iter(args.unstable_features);
 
     configure_color(args.color_choice);
     let mut config = GlobalConfig::new();
+    config.set_log_level(args.check_release.verbosity.log_level());
```

Comment:
This line appears to be duplicated below the big if/elif block. Is that intentional?

Reply:
it's because the verbosity is not a global option, it's in the `CheckRelease` struct, so it could either be in the `SemverChecks.check_release` from `cargo semver-checks -v ...` or in the subcommand `cargo semver-checks check-release -v ...` - currently these are distinct.  The first line is from before we know whether we are running `check_release` (e.g., for bug report or lint info or now unstable help), but we still want to have access to the log level.  We could also move the verbosity field to the global options to get rid of the duplicated lines, but that would be a slightly bigger change.

Reply:
Ah that's interesting and I think unintentional. If you want to open a separate PR to move it to the global scope, I'd happily merge that.